src/subsystems/php/builders/granular-php/default.nix

```diff
@@ -36,26 +37,30 @@
 
     inherit (pkgs.callPackage ../../semver.nix {}) satisfies;
 
-    # php with required extensions
-    php =
-      if satisfies pkgs.php81.version subsystemAttrs.phpSemver
+    phpPackages = l.attrValues inputs.phps.packages.x86_64-linux;
```

Comment:
Probably does not make much difference, maybe passing `system` attribute here instead of hard-coding `x86_64-linux`.